metron-platform/metron-solr/README.md

```diff
@@ -95,3 +95,19 @@ As of now, we have mapped out the Schemas in `src/main/config/schema`.
 Ambari will eventually install these, but at the moment it's manual and
 you should refer to the Solr documentation [https://lucene.apache.org/solr/guide/6_6](here) in general
 and [here](https://lucene.apache.org/solr/guide/6_6/documents-fields-and-schema-design.html) if you'd like to know more about schemas in Solr.
+
+In Metron's Solr DAO implementation, document updates involve reading a document, applying the update and reindexing the whole document.  
```

Comment:
This is a useful description. Can you take it just a bit further and explicitly spell out the final result? Will an update result in 1 doc in the index or 2? ie. does it overwrite the existing indexed doc or insert a new versioned doc?

Reply:
Done